### PR TITLE
add repo callable

### PR DESCRIPTION
**Pull request by hhsnopek**
The power to create and delete Github repos!

### CODE REVIEW

**Comment by hhsnopek** [comment]
I'm gonna update the readme and such before this goes in

**Comment by hhsnopek** [comment]
this is ready for review @BrandonRomano 😄

File: callable.sh [hunk 1]
```diff
@@ -32,9 +32,160 @@ Github__callable_labels(){
...
+    \"team_id\": \"$teams\",
+    \"auto_init\": $autoInit,
+    \"gitignore_template\": \"$gitignore\",
+    \"license_template\": \"$license\"
+}"
+
+        if [[ "$verbose" == true ]]; then
+            curlOpts+='--verbose'
+        else
+            curlOpts+='-s'
```
Comment (BrandonRomano) [inline callable.sh]: Not sure how portable `+=` is... Maybe double check it for me

File: callable.sh [hunk 2]
```diff
@@ -32,9 +32,160 @@ Github__callable_labels(){
...
+        else # real request
+            local res=$(curl -s \
+                "$curlOpts" \
+                -H "Authorization: token $GITHUB_TOKEN" \
+                -H "Content-Type: application/json" \
+                -X POST "$url" \
+                -d "$data")
+
+            if [[ ! $res =~ 2.. ]]; then
+                echo "Failed create your repository."
```
Comment (BrandonRomano) [inline callable.sh]: Maybe some wording issues here 😬

File: callable.sh [hunk 3]
```diff
@@ -32,9 +32,160 @@ Github__callable_labels(){
...
+                echo "Status: $res"
+            else
+                echo "Successfully create your repository."
```
Comment (BrandonRomano) [inline callable.sh]: Here too 🤔

**Review by BrandonRomano — APPROVED** [review]

**Pull request merged**